experience.html

```diff
@@ -1,4 +1,246 @@
 <!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Experience</title>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"/>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="./css/experience.css"/>
+  </head>  
+
+<body>
+  <header class="header">
+    <a href="" class="logo">logo</a>
+    <input class="menu-btn" type="checkbox" id="menu-btn" />
+    <label class="menu-icon" for="menu-btn"><span class="navicon"></span></label>
+    <ul class="menu">
+        <li><a href="./index.html">Home</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="./experience.html">Experience</a></li>
+        <li><a href="./projects.html">Projects</a></li>
+        <li><a href="./education.html">Education</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+  </header>
+  
+<div class="main">
+    <div class="section">
+      <div class="heading">Experience</div>
+      <div class="sub-heading">
+        “The only source of knowledge is experience.”
+      </div>
+    </div>
+
+    <h3>Jobs / Internships</h3>
+    <h6>
+      These are an amazing oppurtunity to boost up your tech stack.
+    </h6>
+
+    <!-- Card -->
+
+    <ul>
+    
+      <li class="card">
+          <a
+              class="featured-image" 
+              style="background-image:url('https://images.unsplash.com/photo-1556761175-b413da4baf72?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=667&q=80')">
```

Comment:
Please don't use external links to fetch assets. Download the images, optimize them and refer them using relative path